api/utils/keys/yubikey.go

```diff
@@ -665,3 +768,99 @@ const (
 	// slower cards (if there are any).
 	signTouchPromptDelay = time.Millisecond * 200
 )
+
+func setPINAndPUKFromDefault(ctx context.Context, yk *piv.YubiKey) (string, error) {
+	isValid := func(pin string) bool {
+		if len(pin) < 6 || len(pin) > 8 {
```

Comment:
Why the max length restriction?

Reply:
```
ykman piv access change-pin
Enter the current PIN: 
Enter the new PIN: 
Repeat for confirmation: 
Usage: ykman piv access change-pin [OPTIONS]
Try 'ykman piv access change-pin -h' for help.

Error: New PIN must be between 6 and 8 characters long.
```

Reply:
Huh, TIL. It's a Yubico limitation then? I'd mention that in a comment, seems arbitrarily small otherwise.